magento/magento2
# Fixed convert to plain text message text

Changed JS Popup Message of New Email Template "Convert to Plain Text".


### Description (*)
I have tested issue in Latest Magento 2.3.3 version setup.
Email Template "Convert to Plain Text" message and Newsletter Template "Convert to Plain Text" message both are you different.
To maintain the consistency I have modified Email Template Message same as Newsletter Template Message.


### Manual testing scenarios (*)

1. Login Admin -> Marketing 
2. Under Communication Section -> Open Email Templates and Newsletter Templates both in different browser tab.
3. Click on Add New Template button on both the views.
4. Click on Convert to Plain Text button.
5. Observe the js Confirmation Popup text of both the views are different.
6. Email Template Message is : Are you sure you want to strip tags 
Newsletter Template Message is : Are you sure that you want to strip all tags?
To maintain the consistency I have modified Email Template Message same as Newsletter Template Message.


### Questions or comments


### Contribution checklist (*)
